apps/box/src/screens/Settings/ComponentGallery/FormDemo.screen.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  FxBox,
+  FxError,
+  FxRadioButton,
+  FxRadioButtonWithLabel,
+  FxSafeAreaBox,
+  FxSpacer,
+  FxText,
+  FxTextArea,
+  FxTextInput,
+} from '@functionland/component-library';
+import { HeaderText } from '../../../components/Text';
+import React from 'react';
+
+enum InputState {
+  default = 'default',
+  disabled = 'disabled',
+  error = 'error',
+}
+
+export const FormDemoScreen = () => {
+  const [inputState, setInputState] = React.useState<InputState>(
+    InputState.default
+  );
+  const [textAreaState, setTextAreaState] = React.useState<InputState>(
+    InputState.default
+  );
+  return (
+    <FxSafeAreaBox marginHorizontal="20" flex={1}>
+      <HeaderText>Form</HeaderText>
+      <FxSpacer marginTop="32" />
+      <FxBox flex={1}>
+        <FxText variant="bodySmallRegular" color="content1">
+          Input Field
+        </FxText>
+        <FxText variant="bodyXSLight" color="content2" marginBottom="8">
+          Input fields are different from text areas in that they are limited to
+          a single line of text.
+        </FxText>
+        <FxTextInput
+          placeholder="Example Description"
+          disabled={inputState === InputState.disabled}
+          error={inputState === InputState.error}
+        />
+        {inputState === InputState.error && <FxError error="Error message" />}
+        <FxSpacer height={8} />
+        <FxRadioButton.Group
```

Comment:
Nice use of radio buttons to switch form state in the demo 👍

Reply:
🙌